packages/matter-node.js-examples/README.md

```diff
@@ -12,6 +12,43 @@ To better show how to use the library we provide some reference implementations
+* **MultiDeviceNodeLegacy**: This example shows how to start multiple Matter nodes on one MatterServer that are run on their own Port number but share MDNS broadcaster and scanner to optimize resources. Each node can be configured via CLI to be a onoff socket or a light and which shell commands are executed for the on and off commands.
+
+## New API Device Examples
+Some of the command line parameters from the legacy examples, like logging, mdns interface are now part of the new environment and can be set by generic command line parameters or also environment variables. They are thats why no longer handled by the example code directly. These are:
+* `--log-level=...` or environment variable `MATTER_LOG_LEVEL` or in code `environment.vars.set('log.level', Level.DEBUG)`. Allowed values are: Level.FATAL ("fatal"), Level.ERROR ("error"), Level.WARN ("warn"), Level.NOTICE ("notice"), Level.INFO ("info"), Level.DEBUG ("debug")
+* `--log-format=...` or environment variable `MATTER_LOG_FORMAT` or `environment.vars.set('log.format', Format.PLAIN)`. Allowed values are: Format.PLAIN ("plain"), Format.HTML ("html"), Format.ANSI ("ansi")
+* `--storage-path=...` or environment variable `MATTER_STORAGE_PATH` or `environment.vars.set('storage.path', "...")` allows to set the storage location. By default, it will be stored in the users home directory in `.matter/...`. If specified, the given path will be used relative to the current working directory.
+* `--storage-clear` or environment variable `MATTER_STORAGE_CLEAR=1` or `environment.vars.set('storage.clear', true)` allows to define if the storage is reset on startup.
+* `--mdns-networkinterface=...` or environment variable `MATTER_MDNS_NETWORKINTERFACE` or `environment.vars.set('mdns.networkInterface', "...")` allows to limit the DNS announcements and scanning to one network interface. By default, all available interfaces are used.
+Additionally, all command line parameters now require to start with two dashes!
+
+* **BridgedDevicesNode**: This example shows how to build a Matter-Bridge to offer multiple OnOff lights and sockets as a bridge. It can be configured via command line and allow to add shell commands to be executed for on and off commands for the numbered devices. More details see below.
+* **ComposedDeviceNode**: This example shows how to build a "simple" composed device where multiple OnOff lights and sockets are combined as one composed device. The devices are all added on root level, so no bridge is used. It can be configured via command line and allow to add shell commands to be executed for on and off commands for the numbered devices. More details see below.
```

Comment:
* **ComposedDeviceNode**: This example shows how to build a "simple" composed device where multiple OnOff lights and sockets are combined as a single device. The devices are all added at the root level, so no bridge is used. It can be configured via command line and allows for specifications of shell commands to be executed when on and off commands are received at the numbered devices. For more details see below.